src/metpy/calc/thermo.py

```diff
@@ -3292,6 +3292,38 @@ def gradient_richardson_number(height, potential_temperature, u, v, vertical_dim
     return (mpconsts.g / potential_temperature) * (dthetadz / (dudz ** 2 + dvdz ** 2))
 
 
+@exporter.export
+@preprocess_and_wrap(
+    wrap_like='temp_bot',
+    broadcast=('temp_bot', 'temp_top')
+)
+@check_units('[temperature]', '[temperature]')
+def scale_height(temp_bot, temp_top):
```

Comment:
Maybe `temperature_bottom` and `temperature_top`? We usually go with full words as much as possible for function argument names.